test/remote_value_tests/remote_value_switch_test.py

```diff
@@ -76,7 +76,7 @@ async def test_process(self):
         )
         assert remote_value.value is None
         await remote_value.process(telegram)
-        assert remote_value.payload is not None
+        assert remote_value.value is not None
```

Comment:
```suggestion
        assert remote_value.telegram is not None
```
the next line is `assert remote_value.value is True` which implies that it is not `None` but the `telegram` attribute is not tested here.